project_timeline/__manifest__.py

```diff
@@ -6,10 +6,11 @@
 {
     "name": "Project timeline",
     "summary": "Timeline view for projects",
-    "version": "10.0.1.0.0",
+    "version": "10.0.1.1.0",
     "category": "Project Management",
     "website": "https://www.tecnativa.com",
     "author": "Tecnativa, "
+              "Doscaal, "
```

Comment:
Please add yourself on README on contributors section, but this minimum change shouldn't is not too significant for making you be part of coauthors.